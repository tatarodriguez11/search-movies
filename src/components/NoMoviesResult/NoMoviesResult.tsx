export function NoMoviesResults() {
  return <p>No se encontraron películas para esta búsqueda</p>;
}
