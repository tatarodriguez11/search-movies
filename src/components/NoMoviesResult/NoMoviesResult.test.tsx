
import { render } from "@testing-library/react";
import { NoMoviesResults } from "./NoMoviesResult";

describe("NoMoviesResults component", () => {
  it("should match the snapshot", () => {
    const { container } = render(<NoMoviesResults />);
    expect(container).toMatchSnapshot();
  });

  it("should render the correct message", () => {
    const { getByText } = render(<NoMoviesResults />);
    expect(
      getByText("No se encontraron películas para esta búsqueda")
    ).toBeInTheDocument();
  });
});
