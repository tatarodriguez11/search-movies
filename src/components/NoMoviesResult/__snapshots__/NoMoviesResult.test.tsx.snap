// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMoviesResults component should match the snapshot 1`] = `
<div>
  <p>
    No se encontraron películas para esta búsqueda
  </p>
</div>
`;
